### PR TITLE
#VFB-149 - Stack Viewer functionality, VFB-158 - Term info thumbnail selection and VFB-159 - Canvas selection

### PR DESCRIPTION
**Pull request by jrmartin**
This PR is for #VFB-149 - Stack Viewer functionality, fix display of labels while hovering and ID loading from stack viewer.
It also implements VFB-158 - Term info thumbnail selection, launches ID of selected thumbnail on General Information of the Term Info. Finally, takes care of VFB-159, canvas mesh selection.

Tested manually and with a local setup. Deployed to codefresh's dev pipeline and ready for testing.

### CODE REVIEW

File: applications/virtual-fly-brain/client/src/components/StackViewerComponent.js [hunk 1]
```diff
@@ -425,11 +435,11 @@ import React from 'react';
                     if (result[j].trim() !== '') {
                       var index = Number(result[j]);
                       if (i !== 0 || index !== 0) { // don't select template
-                        if (index == 0 && !shift) {
-                          if (!isSelected){
+                        if (index == 0 ) {
+                          if (!window.shiftDown){
                             console.log(that.state.label[i] + ' clicked');
```
Comment (ddelpiano) [inline applications/virtual-fly-brain/client/src/components/StackViewerComponent.js:449]: please clear the console.log if done with the development

File: applications/virtual-fly-brain/client/src/components/StackViewerComponent.js [hunk 2]
```diff
@@ -860,7 +874,7 @@ import React from 'react';
     createStatusText: function () {
       if (!this.state.buffer[-1]) {
         var style = {
-          font: '12px Helvetica',
+          font: '8px Helvetica',
```
Comment (ddelpiano) [inline applications/virtual-fly-brain/client/src/components/StackViewerComponent.js]: styling wise this is still way bigger than what we have in prod atm, we can bring it down again to match the label styling we have in prod. Thanks

File: applications/virtual-fly-brain/client/src/components/StackViewerComponent.js [hunk 3]
```diff
@@ -905,6 +919,7 @@ import React from 'react';
           id: nextProps.id,
           serverUrl: nextProps.serverUrl.replace('http:', window.location.protocol).replace('https:', window.location.protocol)
         });
+        console.log("Next label ", nextProps.label)
```
Comment (ddelpiano) [inline applications/virtual-fly-brain/client/src/components/StackViewerComponent.js:931]: remove this please

**Review by ddelpiano — CHANGES_REQUESTED** [review]

File: applications/virtual-fly-brain/client/src/components/StackViewerComponent.js (hunk 1 above)
Comment (jrmartin) [reply applications/virtual-fly-brain/client/src/components/StackViewerComponent.js]: Done

File: applications/virtual-fly-brain/client/src/components/StackViewerComponent.js (hunk 2 above)
Comment (jrmartin) [reply applications/virtual-fly-brain/client/src/components/StackViewerComponent.js]: This one ended up being that the style object itself was not being used, PIXI had a new way of styling their textures , issue fixed in outdated view and font can be adjusted accordingly if not appropriate still.

**Review by ddelpiano — APPROVED** [review]

**Pull request merged**